akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -259,6 +336,8 @@ final class ClusterClient(settings: ClusterClientSettings) extends Actor with Ac
   var contacts = initialContactsSel
   sendGetContacts()
 
+  var contactsPublished = Set.empty[ActorSelection]
```

Comment:
...but we're just keeping track of `contacts`, which is a collection of selections... (I didn't introduce `contacts`).